vivisect/impemu/platarch/amd64.py

```diff
@@ -19,9 +19,9 @@ class Amd64WorkspaceEmulator(v_i_emulator.WorkspaceEmulator, e_amd64.Amd64Emulat
         e_amd64.REG_RDI, e_amd64.REG_R8,  e_amd64.REG_R9,
     ]
 
-    def __init__(self, vw, logwrite=False, logread=False):
+    def __init__(self, vw, logwrite=False, logread=False, stacksize=4096):
```

Comment:
Seeing all the extra argument plumbing almost makes me wonder if the Emulator base class needs a "initStackMemory()" method which can take the stacksize ( an potentially any other params ) and can be called even well after the constructor.  ( would effect fewer APIs, and i kinda worry about having the emulator instance head toward a zillion optional arguments for all the other cases we encounter. )  thoughts?

the initStackMemory mechanism would also potentially allow different arch emu classes to handle initialization slightly differently from each other if needs be.